invokeai/frontend/web/src/features/ui/components/tabs/ModelManager/subpanels/ModelManagerPanel/ModelListItem.tsx

```diff
@@ -9,29 +9,26 @@ import { selectIsBusy } from 'features/system/store/systemSelectors';
 import { addToast } from 'features/system/store/systemSlice';
 import { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
+import { MODEL_TYPE_SHORT_MAP } from 'features/parameters/types/constants';
 import {
   MainModelConfigEntity,
+  LoRAModelConfigEntity,
   useDeleteMainModelsMutation,
+  useDeleteLoRAModelsMutation,
 } from 'services/api/endpoints/models';
 
 type ModelListItemProps = {
-  model: MainModelConfigEntity;
+  model: MainModelConfigEntity | LoRAModelConfigEntity;
   isSelected: boolean;
   setSelectedModelId: (v: string | undefined) => void;
 };
 
-const modelBaseTypeMap = {
-  'sd-1': 'SD1',
-  'sd-2': 'SD2',
-  sdxl: 'SDXL',
-  'sdxl-refiner': 'SDXLR',
-};
-
```

Comment:
Refactored this constant as we already have a bunch of these enumerations in `constants.ts`.